Library/Homebrew/cmd/outdated.rb

```diff
@@ -11,6 +11,10 @@
 #:
 #:    If `--json=`<version> is passed, the output will be in JSON format. The only
 #:    valid version is `v1`.
+#:
+#:    If `--fetch-HEAD` is passed, fetch upstream repository to detect that HEAD
+#:    formula is outdated. Otherwise HEAD-installation is considered outdated if
+#:    new stable or devel version is bumped after that installation.
```

Comment:
> how long does this operation take on e.g. a standard GitHub repo?

Will test, but shouldn't be too time-consuming for GitHubGitDownloadStrategy. However, for GitDownloadStrategy we need to fetch the repo, so it takes some time. How do I understand whether results are OK or not OK when I do some benchmarks?